develop/reference_workarounds.rst

```diff
@@ -32,30 +31,6 @@ multi-tenant use cases. <when_to_use_citus>`
 
 Citus supports all SQL statements in the multi-tenant use-case. Even in the real-time analytics use-cases, with queries that span across nodes, Citus supports the majority of statements. The few types of unsupported queries are listed in :ref:`unsupported` Many of the unsupported features have workarounds; below are a number of the most useful.
 
-.. _pull_push_workaround:
-
-Work around limitations using CTEs
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-When a SQL query is unsupported, one way to work around it is using CTEs, which use what we call pull-push execution.
-
-.. code-block:: sql
-
-  SELECT * FROM ref LEFT JOIN dist USING (id) WHERE dist.value > 10;
```

Comment:
Although this particular query will be supported, are there any other types of queries that would require the CTE workaround? If so, we should keep this section and update the query inside.